rpcs3/Emu/RSX/GL/GLFragmentProgram.cpp

```diff
@@ -68,13 +68,18 @@ void GLFragmentDecompilerThread::insertConstants(std::stringstream & OS)
 
 	}
 
+	OS << "layout(std140, binding = 2) uniform FragmentConstantsBuffer" << std::endl;
+	OS << "{" << std::endl;
 	for (ParamType PT : m_parr.params[PF_PARAM_UNIFORM])
```

Comment:
@vlj why not `const ParamType& PT` (here and other places). (and same for ParamItem, copying is unnecessary)